src/DotLiquid/Strainer.cs

```diff
@@ -103,11 +103,15 @@ public void AddMethodInfo(string rawName, object target, MethodInfo method)
 
         public bool RespondTo(string method)
         {
+            method = Template.NamingConvention.GetMemberName(method);

```

Comment:
Why this change?

Reply:
When registering methods, we use this function to build a static list of methods. PR wants to allow use of aliases, so calling GetMemberName normalizes the name to PascalCase and thus matches the defined list of methods even if used as pascalCase. This change obviously affects all naming conventions, for better or for worse.

Reply:
Well ouch then, that (affects all naming conventions) is a problem right?

Reply:
I'm not sure of the impact, it needs more study. In C# we return name as-is anyways, so it would do nothing (although I would suggest normalizing there too). In Ruby, it would allow replace_first (current behavior) and ReplaceFirst and replaceFirst but NOT replacefirst or rePlaceFirst. I don't see a negative impact (although some negative tests might become positive)

Another option is to extend the interface to have an explicit way to get all alternatives for a given name and register them in the first place. If it's not implemented, it is strict exact match. Maybe an IAliasNamingConvention to maintain backward compatibility?

Reply:
I *think* it's ok to accept more name formats for filters. What happens if the input had content which names clashed with the newly allowed filter name format?

Reply:
If you register class named test and Test, Strainer.Extend calls GetMemberName and will map to the same in ruby (and thus override). C# passes as is and can live side by side currently. This one line won't change that, but changing c# to normalize names will make it behave it like Ruby.